src/main/java/redis/clients/jedis/commands/ConfigBinaryCommands.java

```diff
@@ -0,0 +1,33 @@
+package redis.clients.jedis.commands;
+
+import java.util.List;
+
+public interface ConfigBinaryCommands {
```

Comment:
These are very few in numbers. We can avoid creating a new interface. Put these in `ConfigCommands`.

Reply:
> These are very few in numbers. We can avoid creating a new interface. Put these in `ConfigCommands`.

reverted!